packages/sanity/src/core/validation/inferFromSchema.ts

```diff
@@ -15,5 +15,3 @@ function inferFromSchema(schema: Schema): Schema {
 
   return schema
 }
-
-export default inferFromSchema
```

Comment:
Might be a non issue but: Could these sort of changes (removing `defaults` for this particular instance for example) be considered a breaking change?